src/transformStateWithClones.js

```diff
@@ -7,7 +7,27 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let newState = { ...state };
+
+  actions.forEach((action) => {
+    let clonedState = { ...newState };
```

Comment:
I think you dont need to create **clonedState** varibale as you can use **newState** that you have already created